src/auto/input_stream.rs

```diff
@@ -0,0 +1,170 @@
+        {
+            callback_guard!();
+            unsafe {
+                let mut error = ptr::null_mut();
+                let _ = ffi::g_input_stream_close_finish(_source_object as *mut _, res, &mut error);
+                let result = if error.is_null() { Ok(()) } else { Err(from_glib_full(error)) };
+                let callback: &&(Fn(Result<(), Error>) + Send + Sync + 'static) = transmute(user_data);
+                callback(result);
+            }
+        }
+        let callback = close_async_trampoline;
+        unsafe {
+            ffi::g_input_stream_close_async(self.to_glib_none().0, io_priority, cancellable.0, Some(callback), Box::into_raw(user_data) as *mut _);
+        }
+    }
+
+    fn has_pending(&self) -> bool {
+        unsafe {
+            from_glib(ffi::g_input_stream_has_pending(self.to_glib_none().0))
+        }
+    }
+
+    fn is_closed(&self) -> bool {
+        unsafe {
+            from_glib(ffi::g_input_stream_is_closed(self.to_glib_none().0))
+        }
+    }
+
+    #[cfg(any(feature = "v2_34", feature = "dox"))]
+    fn read_bytes<'a, P: Into<Option<&'a Cancellable>>>(&self, count: usize, cancellable: P) -> Result<glib::Bytes, Error> {
+        let cancellable = cancellable.into();
+        let cancellable = cancellable.to_glib_none();
+        unsafe {
+            let mut error = ptr::null_mut();
+            let ret = ffi::g_input_stream_read_bytes(self.to_glib_none().0, count, cancellable.0, &mut error);
+            if error.is_null() { Ok(from_glib_full(ret)) } else { Err(from_glib_full(error)) }
+        }
+    }
+
+    #[cfg(any(feature = "v2_34", feature = "dox"))]
+    fn read_bytes_async<'a, P: Into<Option<&'a Cancellable>>, Q: Fn(Result<(), Error>) + Send + Sync + 'static>(&self, count: usize, io_priority: i32, cancellable: P, callback: Q) {
+        let cancellable = cancellable.into();
+        let cancellable = cancellable.to_glib_none();
+        let user_data: Box<Box<Fn(Result<(), Error>) + Send + Sync + 'static>> = Box::new(Box::new(callback));
+        extern "C" fn read_bytes_async_trampoline(_source_object: *mut gobject_ffi::GObject, res: *mut ffi::GAsyncResult, user_data: glib_ffi::gpointer)
+        {
+            callback_guard!();
+            unsafe {
+                let mut error = ptr::null_mut();
+                let _ = ffi::g_input_stream_read_bytes_finish(_source_object as *mut _, res, &mut error);
+                let result = if error.is_null() { Ok(()) } else { Err(from_glib_full(error)) };
+                let callback: &&(Fn(Result<(), Error>) + Send + Sync + 'static) = transmute(user_data);
+                callback(result);
+            }
+        }
+        let callback = read_bytes_async_trampoline;
+        unsafe {
+            ffi::g_input_stream_read_bytes_async(self.to_glib_none().0, count, io_priority, cancellable.0, Some(callback), Box::into_raw(user_data) as *mut _);
+        }
+    }
```

Comment:
Seems this function generated wrong, see https://developer.gnome.org/gio/stable/GInputStream.html#g-input-stream-read-bytes-async
Don't understand how user get `glib::Bytes` from it 
cc @antoyo

Reply:
g_input_stream_read_bytes_finish() gives you the GBytes